Automattic/wp-calypso
# Jetpack Onboarding: Remove completely from this world

Some time ago, we built the Jetpack Onboarding in Calypso. This was never used anywhere, though, so it can be removed completely. There's no good reason to keep dead code around anyway 😉 And Calypso could always use a good cleanup of unused pieces.

So, this PR removes Jetpack Onboarding - the entire section and its contextual components, as well as the related state and data layer, and updates any occurrences in the documentation that no longer make sense.

#### Changes proposed in this Pull Request

* Jetpack Onboarding: Remove completely from this world

#### Testing instructions

* Checkout this branch.
* Keep an eye for related errors in your browser console.
* Connect a new Jetpack site and verify it connects properly (you can use https://jurassic.ninja/create to create a disposable Jetpack site):
  * Start from `/jetpack/connect`
  * Start from wp-admin
* Smoke test the Settings for Jetpack sites and verify they work well and unchanged.
* Verify tests pass.
* Manually go through the codebase and verify we're not leaving any instances of the actions/selectors/components that we just removed.
